core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/edit-ema-palette/edit-ema-palette.component.ts

```diff
@@ -33,6 +33,7 @@ import { DotPaletteStore, PALETTE_TYPES } from './store/edit-ema-palette.store';
 })
 export class EditEmaPaletteComponent implements OnInit, OnDestroy {
     @Input() languageId: number;
+    @Input() variantId: string;
```

Comment:
maybe add {required: true} to input

Reply:
A page could not be a variant, so this could be undefined